src/App.vue

```diff
@@ -875,6 +881,14 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // parse draft filing into the store and get the resources
     let resources: ResourceIF
     switch (this.getFilingType) {
+      case FilingTypes.AMALGAMATION:
+        draftFiling = {
+          ...this.buildAmalgamationFiling(),
+          ...draftFiling
+        }
```

Comment:
I'm trying a different way to have an "empty" amalgamation filing object here (compared with incorporation and registration below). The trick is in filing-template-mixin, where I provide fallback values/objects if they are not present in the draft filing.